riscv-c-api.md

```diff
@@ -299,27 +299,37 @@ Each `TARGET-CLONES-ATTR-STRING` defines a distinguished version of the function
 The syntax of `<TARGET-CLONES-ATTR-STRING>` describes below:
 
 ```
-TARGET-CLONES-ATTR-STRING := 'arch=' EXTENSIONS
-                           | 'default'
+TARGET-CLONES-ATTR-STRING := DEFAULT-ATTR-STRING
+                           | ATTR-STRINGS
 
-EXTENSIONS             := <EXTENSION> ',' <EXTENSIONS>
-                        | <EXTENSION>
+ATTR-STRINGS              := ATTR-STRING
+                           | ';' ATTR-STRINGS
 
-EXTENSION              := <OP> <EXTENSION-NAME> <VERSION>
+DEFAULT-ATTR-STRING       := 'default'
 
-OP                     := '+'
+ATTR-STRING               := 'arch=' EXTENSIONS
+                           | 'priority=' DIGIT 
```

Comment:
I don't think this is correct.  Priority without arch isn't allowed per the english description below.

Also, why not priority 15?  Or -1?  In particular, your LLVM patch appears to accept -1.

Reply:
> I don't think this is correct. Priority without arch isn't allowed per the english description below.

Update BNF to let priority only appear when architecture exists.

> Also, why not priority 15? Or -1? In particular, your LLVM patch appears to accept -1.

Update the DIGIT to allow '-'. Does [0-9]+ already support 15?

Reply:
"DIGIT" is not plural which is what caused my misreading.  :)